fenics/include_notebooks.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+git clone https://github.com/pyiron/pyiron_contrib "$HOME"/from_contrib
+cd "$HOME"/from_contrib/
+git checkout FEniCS_job
+cp "$HOME"/from_contrib/notebooks/fenics_tutorial.ipynb "$HOME"/
+cp -r "$HOME"/from_contrib/pyiron_contrib "$HOME"/
```

Comment:
Should not we use the `pip` installation of `pyiron_contrib`

Reply:
With pip install, it is done more properly. I have updated the file `fenics/include_notebooks.sh` accordingly.